scm/multipart.go

```diff
@@ -0,0 +1,26 @@
+package scm
+
+import "mime/multipart"
+
+func (mw *MultipartWriter) Write(f, v string) {
+	if mw.Error == nil {
+		return
+	}
+	if v == "" {
```

Comment:
Should possibly leave a note in the method comment about empty strings not being written as this differs from the mime/multipart behaviour it's wrapping.

Reply:
done